README.rst

```diff
@@ -0,0 +1,143 @@
+===========================
+openwrt-openwisp-monitoring
+===========================
+
+.. image:: http://img.shields.io/github/release/openwisp/openwrt-openwisp-monitoring.svg
+   :target: https://github.com/openwisp/openwrt-openwisp-monitoring/releases
+
+.. image:: https://img.shields.io/gitter/room/nwjs/nw.js.svg?style=flat-square
+   :target: https://gitter.im/openwisp/general
+   :alt: support chat
+
+------------
+
+`OpenWRT <https://openwrt.org/>`_ configuration agent for 
+`OpenWISP Monitoring <https://github.com/openwisp/openwisp-monitoring>`_.
+
+**Want to help OpenWISP?** `Find out how to help us grow here
+<http://openwisp.io/docs/general/help-us.html>`_.
+
+.. image:: http://netjsonconfig.openwisp.org/en/latest/_images/openwisp.org.svg
+  :target: http://openwisp.org
+
+.. contents:: **Table of Contents**:
+ :backlinks: none
+ :depth: 3
+
+Configuration options
+---------------------
+
+UCI configuration options must go in ``/etc/config/monitoring``.
+
+- ``monitored_interfaces``: interfaces that needs to be monitored, defaults to ``*`` for all interfaces.
+- ``interval``: time after which device data should be sent to server, defaults to ``300``
+
+Compiling openwisp-netjson-monitoring
+-------------------------
```

Comment:
the lines below the heading must be of equal length of the line above or some RST parsers will complain.